lib/record_store/record/caa.rb

```diff
@@ -0,0 +1,46 @@
+module RecordStore
+  class Record::CAA < Record
+    attr_accessor :flags, :tag, :value
+
+    LABEL_REGEX = '[a-z0-9](?:-*[a-z0-9])*'
+    DOMAIN_REGEX = /\A#{LABEL_REGEX}(?:\.#{LABEL_REGEX})\z/
```

Comment:
This regex is not good enough.... something like `domain.c` would match. You need to consider at least 2 characters for the TLD.

Reply:
I've followed the [RFC](https://tools.ietf.org/html/rfc6844#section-5.2) for this...

```
   domain = label *("." label)
   label = (ALPHA / DIGIT) *( *("-") (ALPHA / DIGIT))
```

that said, I meant to make this a case-insensitive REGEX

Reply:
I thought the minimum size of a TLD was 2, but haven't really found anything to support that so 👍

Reply:
I believe you're correct... but I'm basing this on the RFC for the record rather than the commonly accepted idea of a domain